game/tests.py

```
@@ -58,7 +64,14 @@ def test_falls_back_to_python_engine_script(self):
 
 
 class BoardViewTest(TestCase):
-    """The board page should load and initialise a session."""
+    """The board page should load and initialise a session for authenticated users."""
+
+    def setUp(self):
+        from django.contrib.auth.models import User
+        self.user = User.objects.create_user(
+            username='testplayer', password='testpass123'
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "password"

(S106)